cmake/Dependencies.cmake

```diff
@@ -96,8 +96,20 @@ if(BUILD_TEST)
     list(APPEND Caffe2_DEPENDENCY_LIBS ${Benchmark_LIBRARIES})
     include_directories(SYSTEM ${Benchmark_INCLUDE_DIRS})
   else()
+    # build as static library and disable install()
+    set(CAFFE2_OLD_BUILD_SHARED_LIBS ${BUILD_SHARED_LIBS})
+    set(BUILD_SHARED_LIBS OFF)
+    macro (install)  # noop
+    endmacro ()
```

Comment:
Suggested here: http://public.kitware.com/pipermail/cmake/2012-April/050023.html.

Necessary because Benchmark doesn't have a `BENCHMARK_INSTALL` option like gloo does ([here](https://github.com/facebookincubator/gloo/blob/0c9766691d77eb54c5a3293bc29621a82424fabb/CMakeLists.txt#L35-L37)).